### PR TITLE
Contributor guide

### PR DESCRIPTION
**Pull request by mbudiu-vmw**
Signed-off-by: Mihai Budiu <mbudiu@vmware.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+You will need to sign a CLA (Contributor License Agreement) to
+contribute code to DDlog under an MIT license.  This is standard.
+
+## Using git to contribute.
+
+Fork the repository using the "fork" button on github, by following these instructions:
+https://help.github.com/articles/fork-a-repo/.  The main step is the following:
+
+```shell
+git remote add upstream https://github.com/vmware/database-stream-processor.git
```
Comment (ryzhyk) [inline CONTRIBUTING.md:28]: wrong repo

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+git rebase -i --autosquash main
+git push --force-with-lease origin my-new-feature
+```
+
+## Running tests
+
+The tests are run by a shell script in the root directory called `test.sh`.
+To run all tests invoke:
+
+> ./test.sh all
```
Comment (ryzhyk) [inline CONTRIBUTING.md:84]: Let's not ask people to run a script that never finishes.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mbudiu-vmw) [reply CONTRIBUTING.md:28]: Copied this from the link you sent. But it is wrong...

File: CONTRIBUTING.md (hunk 2 above)
Comment (ryzhyk) [reply CONTRIBUTING.md:84]: They can run a subset of tests, leaving complete testing to us.  E.g,. they can run `./test.sh simple`.

**Review by ryzhyk — APPROVED** [review]

**Pull request merged**